src/ontology/mondo-ingest.Makefile

```diff
@@ -413,18 +413,17 @@ tmp/merged.owl: tmp/mondo.owl mondo-ingest.owl tmp/mondo.sssom.ttl
 	$(ROBOT) merge -i tmp/mondo.owl -i mondo-ingest.owl -i tmp/mondo.sssom.ttl --add-prefixes config/context.json \
 			 remove --term "http://purl.obolibrary.org/obo/mondo#ABBREVIATION" --preserve-structure false -o $@
 
-tmp/merged.db: tmp/merged.owl
+tmp/%.db: tmp/%.owl
 	@rm -f .template.db
 	@rm -f .template.db.tmp
 	@rm -f tmp/merged-relation-graph.tsv.gz
-	RUST_BACKTRACE=full semsql make $@ -P config/prefixes.csv
+	RUST_BACKTRACE=full semsql make tmp/$*.db -P config/prefixes.csv
 	@rm -f .template.db
 	@rm -f .template.db.tmp
 	@rm -f tmp/merged-relation-graph.tsv.gz
 
-mondo-ingest.db: tmp/merged.db
+merged.db: tmp/merged.db
 	cp $< $@
-.PRECIOUS: mondo-ingest.db
```

Comment:
## Removed `.PRECIOUS` on `merged.db`
I read up again on `.PRECIOUS` and intermediate files, and how they are deleted after a goal is run (conversing with ChatGPT helped a lot). However, `merged.db` should not be considered an intermediate file because it has a goal. Intermediate files are usually/always referenced in the prerequisites as a variable expansion ([example](https://stackoverflow.com/questions/15189704/makefile-removes-object-files-for-no-reason)), not as something that has a goal of its own. If it has a goal of its own, it shouldn't get deleted after running.